src/compiler/core.ts

```diff
@@ -1029,13 +1033,13 @@ namespace ts {
         return a < b ? Comparison.LessThan : Comparison.GreaterThan;
     }
 
-    export function compareStrings(a: string, b: string, ignoreCase?: boolean): Comparison {
+    export function compareStrings(a: string, b: string, locales?: string | string[], options?: Intl.CollatorOptions): Comparison {

```

Comment:
Do we ever use locale? If not I would remove that parameter. 

Also can we just accentSensitivety everywhere? Seems the only place we do not use it is nav bar ordering, but do not think that we need to.. an I missing something?